### PR TITLE
Theme

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
please review
preview: https://deploy-preview-20--ecom-meltling-moments-frontend.netlify.app/

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Alert } from "./components/Alert/alert";
 import { useAlert } from "./context/alert-context";
 import { Router } from "./router/Router";
 import { useAuth } from "./context/auth-context";
+import { useTheme } from "./context/theme-context";
```
Comment (sri-rishi) [inline src/App.jsx]: Suggestion: can make `index.js` file in context folder and export all context file from there

**Review by sri-rishi — APPROVED** [review]
Functionality is working fine, code is also good , feel free to merge

**Review by dev-enforced — APPROVED** [review]
The live preview is working appropriately. 
https://deploy-preview-20--ecom-meltling-moments-frontend.netlify.app/
Good work @HarshnaNHaswani on the logic setup for Theme of Melting Moments!!

File: src/index.css [hunk 2]
```diff
@@ -33,6 +56,8 @@ body {
   -moz-osx-font-smoothing: grayscale;
```
Comment (ashishkatiyar2404) [inline src/index.css:56]: appreciation: good to see that you are covering other browsers also

**Review by ashishkatiyar2404 — APPROVED** [review]
Code quality is good, nice work
I think you can merge

File: src/App.jsx (hunk 1 above)
Comment (HarshnaNHaswani) [reply src/App.jsx]: will do thanks!

**Comment by HarshnaNHaswani** [comment]
Thank you @sri-rishi  @ashishkatiyar2404 @dev-enforced  for PR

**Pull request merged**